api/python/ai/chronon/repo/compile.py

```
@@ -1,312 +1,56 @@
-def _print_warning(string):
-    # print in yellow - \u001b[33m
-    print(f"\u001b[33m{string}\u001b[0m")
+    compile_context = CompileContext()
+    compiler = Compiler(compile_context)
+    results = compiler.compile()
+    return results
 
 
 if __name__ == "__main__":
-    extract_and_convert()
+    compile()
```

Pylint (3.3.7)

[error] No value for argument 'chronon_root' in function call

(E1120)